src/main/java/seedu/address/model/ReadOnlyVolunteer.java

```diff
@@ -11,6 +12,6 @@ public interface ReadOnlyVolunteer {
      * Returns an unmodifiable view of the volunteer list.
      * This list will not contain any duplicate volunteers.
      */
-    ObservableList<Person> getVolunteerList();
+    List<Volunteer> getVolunteerList();
```

Comment:
Is there a reason why this should be `List` instead? 🤔

Reply:
🤔

Reply:
Must have changed it by accident when I changed Person to Volunteer. Will revert it back.